package.json

```diff
@@ -20,13 +20,13 @@
     "ember-auto-focus": "1.0.9",
     "ember-autoresize": "0.6.0",
     "ember-can": "^0.8.4",
-    "ember-cli": "2.13.1",
+    "ember-cli": "~2.14.0-beta.2",
     "ember-cli-app-version": "^3.0.0",
     "ember-cli-autoprefixer": "0.7.0",
-    "ember-cli-babel": "^5.2.4",
+    "ember-cli-babel": "^6.0.0",
```

Comment:
`6.0.0` is the one selected by ember-cli, but we should be safe to upgrade further.